polls/models.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from wagtail.wagtailcore.models import Page
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+from wagtail.wagtailsearch import index
+
+from molo.core.models import LanguagePage
+
+
+class LanguagePage(LanguagePage):
+    LanguagePage.subpage_types = [
+        'core.HomePage',
+        'core.SectionPage',
+        'core.FooterPage',
+        'polls.Question'
+    ]
+
+
+class Question(Page):
```

Comment:
@miltontony could you check the approach being taken here around subclassing a `Page` for a `Question` and a `Choice` further down?